src/NRedisStack/NRedisStack.csproj

```diff
@@ -15,8 +15,9 @@
   </PropertyGroup>
 
   <ItemGroup>
+	<PackageReference Include="NetTopologySuite" Version="2.5.0" />
 	<PackageReference Include="System.Text.Json" Version="7.0.2" Condition="'$(TargetFramework)' == 'netstandard2.0'" />
-	<PackageReference Include="StackExchange.Redis" Version="2.6.96" />
+	<PackageReference Include="StackExchange.Redis" Version="2.6.122" />
```

Comment:
Let's note in the PR that we're upgrading the version of SE.Redis. Ideally this belongs in a separate PR even.

Reply:
I saw that there are projects that do it automatically, maybe we should do it for SE.Redis as well